docs/topics/about-the-user-interface.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="about-the-user-interface_{context}"]
+= About the {WebName}
+
+The {WebName} for the {ProductName} allows a team of users to assess and analayze applications for risks and suitability for migration to hybrid cloud environments on Red Hat OpenShift.
+
+Use the {WebName} to assess and analyze your applications provides insight throughout the adoption process, at both the portfolio and application levels: inventory, assess, analyze, and manage applications for faster migration to OpenShift via the user interface.
```

Comment:
There seems to be something wrong with this sentence. Maybe replace "provides" with "and get"?

Reply:
Fixed.